public/js/sw-page-transition.js

```diff
@@ -76,11 +76,13 @@ const transition = (function() {
       this.log('fetchWithPageTransition: fetch ' + request.url);
       if (this._pagesBeingLoaded.includes(request.url)) {
         this.log('fetchWithPageTransition: returning cached page ' + request.url);
-        return caches.match(request).then(response => {
-          const index = this._pagesBeingLoaded.indexOf(request.url);
-          this._pagesBeingLoaded.splice(index, 1);
-          return response;
-        });
+        return this.openCache()
+          .then(cache => cache.match(request))
+          .then(response => {
+            const index = this._pagesBeingLoaded.indexOf(request.url);
+            this._pagesBeingLoaded.splice(index, 1);
```

Comment:
Not really for this PR but why is `_pagesBeingLoaded` modified here? It looks like this keeps track of "in-flight" requests and so I would expect it to be modified when the response is received, but instead it seems to be modified when `fetchWithPageTransition()` is called a second time with the same request??